model/config.go

```diff
@@ -146,7 +146,7 @@ type IstioRegistry struct {
 	ConfigRegistry
 }
 
-// RouteRules lists all routing rules in a namespace (or all rules if namespace is "")
+// RouteRules lists all unsorted routing rules in a namespace (or all rules if namespace is "")
 func (i *IstioRegistry) RouteRules(namespace string) []*proxyconfig.RouteRule {
```

Comment:
We certainly need to sort route rules by all its fields. I documented precendece since that's how it works now.